sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/IdentityClientIntegrationTests.java

```diff
@@ -51,7 +51,7 @@ public void deviceCodeCanGetToken() {
         Assert.assertNotNull(token.getToken());
         Assert.assertNotNull(token.getExpiresAt());
         Assert.assertFalse(token.isExpired());
-        token = client.authenticateWithUserRefreshToken(new TokenRequestContext().addScopes("https://vault.azure.net/.default"), token).block();
+        token = client.authenticateWithMsalAccount(new TokenRequestContext().addScopes("https://vault.azure.net/.default"), token.getAccount()).block();
```

Comment:
These tests could be using StepVerifier rather than block.
```java
// Arrange
var context = new TokenRequestContext().addScopes("https://vault.azure.net/.default");

// Act & Assert
StepVerifier.create(client.authenticateWithMsalAccount(context, token.getAccount()))
    .assertNext(token -> {
        assertNotNull(token.getToken());
        assertNotNull(token.getExpiresAt());
    })
    .verifyComplete();
```

Reply:
This is out of scope - if this PR stays open next week I can fix these. But for now I'm planning to get some sleep and enjoy my day off 😸